base/replicator_test.go

```diff
@@ -18,38 +18,26 @@ func TestReplicator(t *testing.T) {
 		Lifecycle: sgreplicate.CONTINUOUS,
 	}
 	_, err := r.Replicate(params, false)
-	goassert.Equals(t, err, nil)
-	waitForActiveTasks(t, r, 1)
+	assert.NoError(t, err)
+	goassert.Equals(t, len(r.ActiveTasks()), 1)
```

Comment:
I think this might've been accidental. Should retain the `waitForActiveTasks(t, r, 1)` behaviour.

Reply:
Yeah I think that was a merge/rebase mistake - I'll revert that.